tool/teleport/common/teleport.go

```diff
@@ -1017,6 +1022,27 @@ func onIntegrationConfEICEIAM(params config.IntegrationConfEICEIAM) error {
 	return nil
 }
 
+func onIntegrationConfAWSAppAccessIAM(params config.IntegrationConfAWSAppAccessIAM) error {
+	ctx := context.Background()
+
+	// Ensure we print output to the user. LogLevel at this point was set to Error.
+	utils.InitLogger(utils.LoggingForDaemon, slog.LevelInfo)
+
+	iamClient, err := awsoidc.NewAWSAppAccessConfigureClient(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	err = awsoidc.ConfigureAWSAppAccess(ctx, iamClient, awsoidc.AWSAppAccessConfigureRequest{
+		IntegrationRole: params.Role,
+	})
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	return nil
```

Comment:
Suggestion: return directly? trace.Wrap has the `if err != nil` built into it anyway.

```suggestion
	return trace.Wrap(err)
```